plexapi/video.py

```diff
@@ -622,3 +643,31 @@ def show(self):
     def _defaultSyncTitle(self):
         """ Returns str, default title for a new syncItem. """
         return '%s - %s - (%s) %s' % (self.grandparentTitle, self.parentTitle, self.seasonEpisode, self.title)
+
+    def subtitleStreams(self):
+        """ Returns a list of :class:`~plexapi.media.SubtitleStream` objects for all MediaParts. """
+        streams = []
+        for elem in self.media:
```

Comment:
use iterParts()

Reply:
This was a copy paste from the same function found in[ Movie class.](https://github.com/pkkid/python-plexapi/blob/master/plexapi/video.py#L222-L228). Want me to correct both? Or correct it once and place it in the Video class since both Movie and Episode load the Video class.

Reply:
In the latest commit this function and the other subtitle related functions have been moved to the Video class.